README.md

```diff
@@ -1,6 +1,53 @@
-# FINDOLOGIC export toolkit
+FINDOLOGIC export toolkit
```

Comment:
Use Markdown headings with hash prefixes, because the `<hX>` level is more obvious from the number of hashes, as compared to special syntax required by "underline" syntax.

Reply:
Done